server.go

```diff
@@ -206,6 +203,18 @@ func (srv *Server) Serve(l net.Listener) error {
 	}
 }
 
+func (srv *Server) SetChannelHandler(kind string, handler ChannelHandler) {
+	srv.ensureHandlers()
+	srv.mu.Lock()
+	defer srv.mu.Unlock()
+	srv.channelHandlers[kind] = handler
+}
+
+func (srv *Server) GetChannelHandler(kind string) ChannelHandler {
```

Comment:
It's more idiomatic to drop the "Get" in getters in Go.

Reply:
I've updated the PR.